.circleci/config.yml

```diff
@@ -150,11 +150,11 @@ jobs:
       - run:
           name: Build
           working_directory: ~/project/packages/vm
-          command: cargo build --locked --no-default-features --features "default-cranelift iterator"
+          command: cargo build --locked --no-default-features --features default-cranelift,iterator
       - run:
           name: Test
           working_directory: ~/project/packages/vm
-          command: cargo test --locked --no-default-features --features "default-cranelift iterator"
+          command: cargo test --locked --no-default-features --features default-cranelift,iterator
```

Comment:
Why this diff?

```
$ cargo test --help
cargo-test 
Execute all unit and integration tests and build examples of a local package

USAGE:
    cargo test [OPTIONS] [TESTNAME] [-- <args>...]

OPTIONS:
    // ...
        --features <FEATURES>...     Space-separated list of features to activate
```

Reply:
ah, in every example I have seen, they have been comma-separated, I found the quotes a bit odd. But if this is the standard format suggested by the cli, I will revert this

Reply:
okay, I don't mind. If we don't need the quotes, that's also nice